src/realmCryo.ttl

```diff
@@ -50,57 +52,133 @@ sorelsp:inside rdf:type owl:ObjectProperty .
 ###  http://sweetontology.net/realmCryo/AblationZone
 soreac:AblationZone rdf:type owl:Class ;
                   rdfs:subClassOf soreac:GlacialRegion ;
-                  rdfs:label "ablation zone"@en .
+                  rdfs:label "ablation zone"@en ;
+                  skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_01000914> ;
+                  skos:definition  [
+                   rdfs:comment  "An environmental zone which overlaps a glacier or an ice sheet, and in which ice loss exceeds ice gain."@en ;
+                   dcterms:source <https://en.wikipedia.org/wiki/Ablation_zone> ;
+                   dcterms:created "2019-12-10T06:11:13-08:00Z"^^xsd:dateTimeStamp ;
+                   dcterms:creator <https://orcid.org/0000-0003-4091-6059> ;
+                   prov:wasDerivedFrom <http://purl.obolibrary.org/obo/ENVO_01000914> ;
+             ] .
 
 
 ###  http://sweetontology.net/realmCryo/AccumulationZone
 soreac:AccumulationZone rdf:type owl:Class ;
                       rdfs:subClassOf soreac:GlacialRegion ;
-                      rdfs:label "accumulation zone"@en .
+                      rdfs:label "accumulation zone"@en ;
+                      skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_01001352> ;
+                      rdfs:comment  "This zone usually occurs at higher elevations and generally overlaps the conversion of snow to glacial ice."@en ;
+                      skos:definition  [
+                        rdfs:comment  "An ice accumulation zone which overlaps part of a glacier."@en ;
+                        dcterms:source <https://orcid.org/0000-0003-4808-4736> ;
+                        dcterms:created "2019-12-10T06:11:13-08:00Z"^^xsd:dateTimeStamp ;
+                        dcterms:creator <https://orcid.org/0000-0003-4091-6059> ;
+                        prov:wasDerivedFrom <http://purl.obolibrary.org/obo/ENVO_01001352> ;
+                      ] .
 
 
 ###  http://sweetontology.net/realmCryo/AlpineTundra
 soreac:AlpineTundra rdf:type owl:Class ;
                   rdfs:subClassOf soreac:Tundra ;
-                  rdfs:label "alpine tundra"@en .
+                  rdfs:label "alpine tundra"@en ;
+                  skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_3400001> ;
+                  skos:exactMatch <http://purl.obolibrary.org/obo/ENVO_3400001> ;
+                  skos:definition  [
+                        rdfs:comment  "An area of tundra which is present in a region subject to alpine conditions."@en ;
```

Comment:
No it is not just tundra in mountains.  Yes, the SWEET hierarchy is screwed up; but until the ENVO alpine tundra biome, ecosystem, and other such terms are harmonized this is as best as we can do.  As you might expect there are two GCW definitions which are in conflict...

1. A form of tundra in which the absence of trees is due to high altitude instead of high latitude. It lies roughly between the summer isotherm of 10C and the snow line. (Also called mountain tundra.)

2. High altitude biome dominated by a few species of dwarf shrubs, a few grasses, sedges, lichens, and mosses. Productivity is low in this biome because of the extremes of climate. Quite similar to tundra.

Reply:
Why not use the GCW definition?  the "It lies roughly between the summer isotherm of 10C and the snow line. This high altitude biome is dominated by a few species of dwarf shrubs, a few grasses, sedges, lichens, and mosses. Productivity is low in this biome because of the extremes of climate." part should be a scope note not a defintion.

Reply:
Collective comments from our [Semantic Harmonization call](
https://docs.google.com/document/d/1IFA6AsCdxESh9mZB5W1bZ_vARFkF3OvnL-KgdGNcp60/edit#heading=h.2uotl4egsd38)

Noted, once again, that this is a SWEET revision issue rather than harmonisation issue. 

Separate issues should be created for definition and hierarchy issues, noting that a policy on definitions in SWEET should precede focused work. 

Fixed mapping for the PR

Reply:
definition is from ENVO